### PR TITLE
Support the option to exclude versions when generating delivery urls

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: test/spec/cloudinary-spec.js [hunk 1]
```diff
@@ -170,6 +170,10 @@ describe('Cloudinary', function() {
       shorten: true
     }, protocol + '//res.cloudinary.com/test123/iu/test', {});
   });
+  it('Should not set default version v1 to resources stored in folders and ignore the version parameter if exclude_version is set to true', function() {
+    test_cloudinary_url('folder/test', {exclude_version:true}, protocol + '//res.cloudinary.com/test123/image/upload/folder/test', {exclude_version:true});
```
Comment (asisayag2) [inline test/spec/cloudinary-spec.js]: @maoznir , isn't that test needs to check with exclude_version=false (or without it altogether)?

**Comment by asisayag2** [comment]
@maoznir , exclude_version is a URL configuration, like many others. 
It needs to be mentioned in more places. 
Please search for other URL config items (like private_cdn for example) and consider where it should be added

File: test/spec/cloudinary-spec.js (hunk 1 above)
Comment (asisayag2) [reply test/spec/cloudinary-spec.js]: @maoznir ?

File: src/url.js [hunk 2]
```diff
@@ -196,6 +196,6 @@ export default function url(publicId, options = {}, config = {}) {
   }
   prefix = cloudinaryUrlPrefix(publicId, options);
   resourceTypeAndType = finalizeResourceType(options.resource_type, options.type, options.url_suffix, options.use_root_path, options.shorten);
-  version = options.version ? 'v' + options.version : '';
+  version = (options.version && !options.exclude_version) ? `v${options.version}` : '';
```
Comment (tocker) [inline src/url.js]: Should be `force_version`

**Comment by tocker** [comment]
Replaced by #173

**Pull request closed without merging**